clang/test/CodeGenSYCL/stdtypes_kernel_type.cpp

```diff
@@ -1,29 +1,6 @@
-// RUN: %clang_cc1 -fsycl -fsycl-is-device -fsycl-int-header=%t.h -DCHECK_ERROR -verify %s
-// RUN: %clang_cc1 -fsycl -fsycl-is-device -triple spir64-unknown-unknown-sycldevice -fsycl-int-header=%t.h %s
-// RUN: FileCheck -input-file=%t.h %s
-//
-// CHECK: #include <CL/sycl/detail/defines_elementary.hpp>
-// CHECK-NEXT: #include <CL/sycl/detail/kernel_desc.hpp>
-//
-// CHECK: static constexpr
-// CHECK-NEXT: const char* const kernel_names[] = {
-// CHECK-NEXT:   "_ZTSm",
-// CHECK-NEXT:   "_ZTSl"
-// CHECK-NEXT: };
-//
-// CHECK: static constexpr
-// CHECK-NEXT: const kernel_param_desc_t kernel_signatures[] = {
-// CHECK-NEXT:   //--- _ZTSm
-// CHECK-EMPTY:
-// CHECK-NEXT:   //--- _ZTSl
-// CHECK-EMPTY:
-// CHECK-NEXT: };
-
-// CHECK: template <> struct KernelInfo<unsigned long> {
-// CHECK: template <> struct KernelInfo<long> {
-
-void usage() {
-}
+// RUN: %clang_cc1 -fsycl -fsycl-is-device -sycl-std=2020 -DCHECK_ERROR -verify %s
```

Comment:
Since we don't check integration header generation in this test anymore, can we move it to SemaSYCL directory?